docs/user-guide/api-mediation/configuration-logging.md

```diff
@@ -0,0 +1,104 @@
+# Customizing Zowe API Mediation Layer logging
+
+Zowe API Mediation Layer uses logback for its logging. You can customize the logging of Zowe API Mediation Layer by specifying the customized `logback.xml` file in `zowe.yaml` for each service separately.
+
+To change the default logback configuration file, set `components.<component>.logging.config` with a path to your `logback.xml`.
+
+* `component`  
+  Specifies one of the following services:
+    - `gateway`
+    - `discovery`
+    - `api-catalog`
+    - `caching-service`
+    - `cloud-gateway`
+    - `metrics-service`
+
+**Example with Gateway Service:**
+
+```yaml
+components:
+  gateway:
+    logging:
+      config: /path/to/logback.xml
```

Comment:
Do we know the root folder to use a relative address?

Reply:
No, it could be anything, that's why it's usually written that way (if you will take a look at other examples, like this one: https://docs.zowe.org/stable/user-guide/install-zowe-zos-convenience-build/#step-4-copy-the-zoweyaml-configuration-file-to-preferred-location)